src/main/java/org/opensearch/security/dlic/rest/api/AbstractApiAction.java

```diff
@@ -1,16 +1,16 @@
 /*
- * Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
Sorry, the latest one should be [this](https://github.com/opensearch-project/security/blob/main/src/main/java/org/opensearch/security/setting/TransportPassiveAuthSetting.java#L1-L14).

Reply:
No changes to the existing copyright header except for year update (optional)